docs/general/xs/bugs/bugs.json

```diff
@@ -513,5 +513,20 @@
             "Version Fixed": "153015",
             "show": false
         }
+    ],
+    "cSetTechCost": [
```

Comment:
Use key `"Individual Tech Modifiers"` here please. This is reused as the file name.